os-autoinst-obs-auto-submit

```diff
@@ -93,9 +93,9 @@ update_package() {
     sed -i -e 's/^  \* $/  * Update to latest openQA version/' ./*.changes
     $osc ci -m "Offline generation of ${version}" $ci_args
     rc=0
+    wait_for_package_build "$target"
+    cmd="$osc sr"
```

Comment:
IIUC, in line 102 `cmd="$cmd -s $req"`, so with every loop it gets extended.
That's why this needs to stay in the loop I think.

Reply:
Not sure I follow. whats extended? you mean `cmd` or `$req`. `$cmd` is always `osc sr` and $req indeed is updated with every loop, but it is already in that loop